modules/precisonatBidAdapter.md

```diff
@@ -0,0 +1,74 @@
+# Overview
+
+```
+Module Name: Precisonat Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: tech@preciso.net
+```
+
+# Description
+
+Module that connects to preciso' demand sources
+please reach out to tech@preciso.net to receive your own publisher id
+
+# Parameters
+
+| Name          | Scope    | Description               | Example              |
+| :------------ | :------- | :------------------------ | :------------------- |
+| `publisherId` | required (for prebid.js) | partner ID provided by preciso | PreTest_0001
+| `region`   | required (for prebid.js)     | 3 letter country code | "USA" |
```

Comment:
Region can't be required. It's difficult for global publishers to set this kind of metadata on their cached pages around the globe. You should be sniffing the IP address to get a sense for where in the world this user is.